lms/static/sass/partials/lms/theme/_variables-v1.scss

```
@@ -193,6 +193,8 @@ $google-blue: #4285f4;
 $google-focus-blue: #287ae6;
 $microsoft-black: #2f2f2f;
 $microsoft-focus-black: #000;
+$apple-black: #000000;
+$apple-focus-black: #000000;
```

If both colors are same, you can simply do this:
```suggestion
$apple-focus-black: $apple-black;
```